### PR TITLE
Feat/support on chain asset registry

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Have added support to on-chain asset registry.. Have removed a lot of code from GA.. Please review and let me know your feedback..

### CODE REVIEW

File: packages/types/src/extrinsic/types.ts [hunk 1]
```diff
@@ -102,5 +104,6 @@ declare module '@polkadot/types/interfaceRegistry' {
     ChargeTransactionPayment;
     FeeExchange: FeeExchange;
     'Option<FeeExchange>': Option<FeeExchange>;
+    'Vec<(T::AssetId, AssetInfo)>': Vec<ITuple<[AssetId, AssetInfo]>>;
```
Comment (holygits) [inline packages/types/src/extrinsic/types.ts:107]: Could be nice to alias `(T::AssetId, AssetInfo)` as `AssetMeta` or something like that so the type carries more meaning

File: packages/api/src/rpc/genericAsset.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {RpcMethodOpt} from '@polkadot/jsonrpc/types';
+
+import createMethod from '@polkadot/jsonrpc/create/method';
+
+const registeredAssets: RpcMethodOpt = {
+  description: 'RPC call for querying asset metadata',
```
Comment (holygits) [inline packages/api/src/rpc/genericAsset.ts:6]: maybe something like: 'Get all registered generic assets (symbol, decimal places)'

File: packages/api/test/e2e/queries.e2e.ts [hunk 3]
```diff
@@ -128,6 +128,7 @@ describe('e2e queries', () => {
        expect(registeredAsset.length).toBeGreaterThan(0);
        const [cpayAssetId, cpayAssetInfo] = registeredAsset[0];
        const [cennzAssetId, cennzAssetInfo] = registeredAsset[1];
+       expect(true).toBe(false);
```
Comment (holygits) [inline packages/api/test/e2e/queries.e2e.ts:131]: 🤣

Comment (KarishmaBothara) [reply packages/api/test/e2e/queries.e2e.ts]: 😂

File: packages/types/src/extrinsic/types.ts [hunk 4]
```diff
@@ -102,5 +105,8 @@ declare module '@polkadot/types/interfaceRegistry' {
     ChargeTransactionPayment;
     FeeExchange: FeeExchange;
     'Option<FeeExchange>': Option<FeeExchange>;
+    // AssetMetaTuple: AssetMetaTuple;
```
Comment (holygits) [inline packages/types/src/extrinsic/types.ts:108]: After remove this stuff lgtm!

**Pull request merged**